exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -289,6 +289,10 @@ private ExecConstants() {
 
   public static final OptionValidator COMPILE_SCALAR_REPLACEMENT = new BooleanValidator("exec.compile.scalar_replacement");
 
+  // Controls whether to enable bulk parquet reader processing
+  public static final String PARQUET_FLAT_READER_BULK                    = "store.parquet.flat.reader.bulk";
```

Comment:
Could you please fix shift between variable name and equals? Looks like it correspond to the general code style in Drill project. Please consider the same in other changed classes.

Reply:
A mentor of mine insisted on code formatting to improve the overall code readability; it is very hard for me to forgo  such a practice.. but I did. I went through the changes and removed all formatting around the equal sign.

Reply:
While it marginally improves code readability, maintaining the code style that aligns on the longest line is not sustainable as it requires modifications to existing code when a new line is introduced or otherwise code becomes unreadable. My vote is to avoid newly introduced code formatting as well and keep existing Drill style.